POLMED/src/components/button/index.tsx

```diff
@@ -0,0 +1,10 @@
+type Props = {
+    content: string;
+    className: string;
+};
+
+const Button = ({content, className}:Props) => {
+    return <button className={className}>{content}</button>;
```

Comment:
Proponuje dodawać tylko klasy, które powyżej wyszczególniłem. Wiemy, że jest to przycisk więc możemy z góry ustawić klasę _btn_.
![image](https://user-images.githubusercontent.com/33033063/205491451-fc8b15be-d1d5-4ede-904a-2e2a086cde9b.png)